scripts/build-tools.sh

```diff
@@ -5,27 +5,71 @@
 # fail immediately on any errors
 set -e
 
-cd tools
-
-rm -rf build_tools
-
-mkdir build_tools
-cd build_tools
-cmake ..
-make -j$(nproc)
-for args in $@
-do
-	#build test topologies
-	if [[ "$args" == "-t" ]]; then
-		make tests -j$(nproc)
-	fi
-	#build fuzzer
-	if [[ "$args" == "-f" ]]; then
-		rm -rf build_fuzzer
-		mkdir build_fuzzer
-		cd build_fuzzer
-		cmake ../../fuzzer
-		make -j$(nproc)
-		cd ../
-	fi
-done
+SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
+SOF_REPO=$(dirname "$SCRIPT_DIR")
+NO_PROCESSORS=$(nproc)
+
+print_usage()
+{
+        echo "usage: $0 [-t|-f]"
+        echo "       [-t] Build test topologies"
+        echo "       [-f] Build fuzzer"
```

Comment:
Nit:
```shell
cat <<EOFUSAGE
usage: $0 [-t|-f]
             [-t] Build test topologies
             [-f] Build fuzzer"
EOFUSAGE
```
FYI https://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo